pkg/tinyvcdiff/include/vcdiff_mtd.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2022 Juergen Fitschen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @defgroup pkg_tinyvcdiff   Tiny VCDIFF
+ * @ingroup  pkg
```

Comment:
Duplicated from doc.txt (maybe a bug in the generator ? )

```suggestion
 * @ingroup  pkg_tinyvcdiff
```

Reply:
Good catch! Thank you. Yes, too much copy pasta involved ;-)